docs/src/man/comparisons.md

```diff
@@ -0,0 +1,27 @@
+The table compares the main functions of DataFrames with R dplyr and Stata
+| DataFrames       | dplyr | Stata|
+|:------------|:------------|:------------|
+|`combine(df, :x => mean)`  | `summarize(mean(x))`    | `collapse (mean) x `|
```

Comment:
I don't think ungrouped sumaries are very common. Better to show off `describe` from data frames and `summary` from stata?

Reply:
Adding `groupby` for each command makes it too verbose. To clarify, I've re-arranged the table after the paragraph explaining how to use `groupby`.